Add example repo to spring-boot-spa post

bdemers opened the pull request:
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [ ] A title that's approved by Dev Advocacy
- [ ] A URL that's approved by Dev Advocacy
- [ ] The content has been ran through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist

bdemers committed:
Add example repo to spring-boot-spa post

mraible committed:
Merge branch 'main' into spring-boot-spa-repo

mraible commented on _source/_posts/2021-10-04-spring-boot-spa.md:
```diff
@@ -11,6 +11,8 @@ tweets:
 image: blog/spring-boot-spa/spring-boot-spa-social.jpg
 type: conversion
+changelog:
+- 2019-10-05: Created example [GitHub repository](https://github.com/oktadev/okta-vuejs-spa-spring-boot-example).
```
```suggestion
github: https://github.com/oktadev/okta-vuejs-spa-spring-boot-example
```

mraible commented on _source/_posts/2021-10-04-spring-boot-spa.md:
```diff
@@ -11,6 +11,8 @@ tweets:
+- 2019-10-05: Created example [GitHub repository](https://github.com/oktadev/okta-vuejs-spa-spring-boot-example).
 ---
 
 In this tutorial, you are going to create a single-page application (SPA) that uses a Spring Boot resource server and a Vue front-end client. You'll see how to configure Spring Boot to use JSON Web Tokens (JWT) for authentication and authorization, with Okta as an OAuth 2. 0 and OpenID Connect (OIDC) provider. You'll also see how to bootstrap a Vue client app with the Vue CLI and how to secure it ...
```
```suggestion
In this tutorial, you are going to create a single-page application (SPA) that uses a Spring Boot resource server and a Vue front-end client. You'll see how to configure Spring Boot to use JSON Web Tokens (JWT) for authentication and authorization, with Okta as an OAuth 2.0 and OpenID Connect (OIDC) provider. You'll also see how to bootstrap a Vue client app with the Vue CLI and how to secure it using the Okta Sign-In Widget.
```

bdemers committed:
Apply suggestions from code review

Co-authored-by: Matt Raible <matt.raible@okta.com>

bdemers committed:
Merge branch 'main' into spring-boot-spa-repo

bdemers committed:
Update spring-boot-spa post:

- pin NPM versions
- correct typo (you/your)
- move github repo link to bottom of post (out of changelog)

bdemers commented on _source/_posts/2021-10-04-spring-boot-spa.md:
```diff
@@ -248,10 +249,10 @@ You need to install a few dependencies. The first three are required for Okta se
+@okta/okta-signin-widget@5.12.0 \
+@okta/okta-vue@5.0.1 \
+@okta/okta-auth-js@5.5.0 \
+axios@0.22.0
```
```suggestion
  @okta/okta-signin-widget@5.12.0 \
  @okta/okta-vue@5.0.1 \
  @okta/okta-auth-js@5.5.0 \
  axios@0.22.0
```

bdemers committed:
Update _source/_posts/2021-10-04-spring-boot-spa.md
